iron-menu-behavior.html

```diff
@@ -270,7 +302,9 @@
      * @param {CustomEvent} event A key combination event.
      */
     _onDownKey: function(event) {
-      this._focusNext();
+      if (this._getClosestItem(event.detail.keyboardEvent)) {
```

Comment:
This kind of seems like an inversion of what `stopPropagation` would do for us. Perhaps we need to re-consider that approach. WDYT?